packages/@okta/vuepress-theme-prose/layouts/Layout.vue

```diff
@@ -5,19 +5,19 @@
     </div>
 
     <div class="page-body">
-      <Breadcrumb />
+      <Breadcrumb/>
       <div class="content" v-if="$page.frontmatter.component">
         <component :is="$page.frontmatter.component" />
       </div>
       <div class="content" v-else>
         <div class="content--container">
           <div class="tree-nav">
-            <Sidebar />
+            <Sidebar :sidebarActive="treeNavOpen"/>
           </div>
           <div class="content-area">
             <PageTitle />
             <MobileOnThisPage />
-            <Content />
+            <Content/>
```

Comment:
To keep with consistency, lets add the space back in here. Typically, self closing tags in Vue have `<Component />`